packages/flame/test/components/component_test.dart

```diff
@@ -1,3 +1,6 @@
+import 'dart:async';
```

Comment:
```suggestion
```

Reply:
how come analyzer is not catching that???